usr/actor.c

```diff
@@ -67,7 +67,7 @@ actor_delete(actor_t *thread)
 		/* priority: low */
 		/* fallthrough */
 	case ACTOR_SCHEDULED:
-		log_debug(1, "deleting a scheduled/waiting thread %p:%s!",
+		log_debug(4, "deleting a scheduled/waiting thread %p:%s!",
```

Comment:
I would put this in a different patch and note why it changed. I think Chris or someone (maybe me) made it a 1 because it shouldn't happen often. However, I think something changed and we see it all the time now.